pkgs/build-support/setup-hooks/compress-man-pages.sh

```diff
@@ -6,10 +6,13 @@ compressManPages() {
     if [ ! -d "$dir/share/man" ]; then return; fi
     echo "gzipping man pages in $dir"
 
-    GLOBIGNORE=.:..:*.gz:*.bz2
-
+    # compress all uncompressed manpages
     for f in "$dir"/share/man/*/* "$dir"/share/man/*/*/*; do
-        if [ -f "$f" -a ! -L "$f" ]; then
+        if [ "${f: -3}" != ".gz" \
```

Comment:
Why not use `=~` here? It feels awkward to have to mention the length of the right string to strip off that component on the left